### DISCUSSION TITLE
System compactability

### DISCUSSION THREAD

**Discussion by luciyaroshni** [apache/skywalking#8381] [Q&A]
Could you please inform us what all OS will support Skywalking and is there any specific system dependency is there.  Is there any minimum and maximum specification is there? 

We have tested in RHEL 7, Centos 7 and 8. Could you please tell us if this will support any higher version or in windows?

**Comment by wu-sheng** [comment]
As a Java application, we don't care the OS version and vendor much.

**Reply by wu-sheng** [reply]
Once you have JVM 9+ up and running, you should be good.
